### PR TITLE
LIVY-163. Fixed missing Python traceback in interactive mode.

### PR DESCRIPTION
**Pull request by tc0312**
- Also stripped the frame of fake_shell in the traceback. It should be transparent to users.

### CODE REVIEW

File: repl/src/test/scala/com/cloudera/livy/repl/PythonInterpreterSpec.scala [hunk 1]
```diff
@@ -150,6 +149,7 @@ class PythonInterpreterSpec extends BaseInterpreterSpec {
       "name 'x' is not defined",
       List(
         "Traceback (most recent call last):\n",
+        "  File \"<stdin>\", line 1, in <module>\n",
```
Comment (vanzin) [inline repl/src/test/scala/com/cloudera/livy/repl/PythonInterpreterSpec.scala]: I wonder how useful this is? Is there a proper correlation between the line shown here and anything the user has submitted?

From what I remember there is not a 1:1 relationship between a statement submission and a "line" sent to the interpreter.

Comment (tc0312) [inline repl/src/test/scala/com/cloudera/livy/repl/PythonInterpreterSpec.scala]: The stack trace is useful when users are using some complicated library (Pandas) and it returns an error.

Comment (vanzin) [inline repl/src/test/scala/com/cloudera/livy/repl/PythonInterpreterSpec.scala]: Sorry, I was going to clarify. The stack trace is ok, I meant the frames mentioning `<stdin>` and line numbers.

Comment (tc0312) [inline repl/src/test/scala/com/cloudera/livy/repl/PythonInterpreterSpec.scala]: Changed the clean up. Traceback of
```python
def func1():
  raise Exception("message")
def func2():
  func1()
func2()
```
will become
```python
Traceback (most recent call last):
  File "<stdin>", line 4, in func2
  File "<stdin>", line 2, in func1
Exception: message"
```

I don't want to do too many changes at once and will leave line numbers as is. We can revisit line number later.

**Comment by vanzin** [comment]
Merging.

**Pull request closed without merging**